R/logger.R

```diff
@@ -67,11 +67,50 @@ fallback_namespace <- function(namespace) {
     namespace
 }
 
+#' Base Logging Function
+#' @param fun_name string a full name of log function
+#' @param arg see \code{\link{log_levels}}
+#' @param namespace logger namespace
+#' @param index index of the logger within the namespace
+#' @return currently set or return log function property
+#' @keywords internal
+log_base <- function(fun_name, arg, namespace, index) {
```

Comment:
I'm thinking about renaming to something like `log_config_setter` -- thoughts?

Reply:
`log_config_setter` sounds nice, applied